### ISSUE TITLE
Math functions not correctly exported (ComputeCpp CE 1.0.3 - ComputeCpp_vs2015_d.lib)

### ISSUE THREAD

**Issue by rosenrodt** [codeplaysoftware/computecpp-sdk#164]
I am building the samples from the computecpp-sdk using VS2015 and ComputeCpp CE 1.0.3

Here is what I got when linking against ComputeCpp_vs2015_d.lib. It seems some symbols are not exported from the library. And it's all related to `abacus` math functions

```
Error	LNK2019	unresolved external symbol "__declspec(dllimport) unsigned __int64 __cdecl abacus::min(unsigned __int64,unsigned __int64)" (__imp_?min@abacus@@YA_K_K0@Z) referenced in function "unsigned __int64 __cdecl cl::sycl::min<unsigned __int64,0>(unsigned __int64,unsigned __int64)" (??$min@_K$0A@@sycl@cl@@YA_K_K0@Z)	scan	D:\ThirdPartySrc\computecpp-sdk\build\samples\scan\scan.obj	1	

Error	LNK2019	unresolved external symbol "__declspec(dllimport) float __cdecl abacus::pow(float,float)" (__imp_?pow@abacus@@YAMMM@Z) referenced in function "float __cdecl cl::sycl::pow<float,float,float,0>(float,float)" (??$pow@MMM$0A@@sycl@cl@@YAMMM@Z)	opencl-c-interop	D:\ThirdPartySrc\computecpp-sdk\build\samples\opencl-c-interop\opencl-c-interop.obj	1	

Error	LNK2019	unresolved external symbol "__declspec(dllimport) float __cdecl abacus::exp(float)" (__imp_?exp@abacus@@YAMM@Z) referenced in function "float __cdecl cl::sycl::exp<float,float,0>(float)" (??$exp@MM$0A@@sycl@cl@@YAMM@Z)	gaussian-blur	D:\ThirdPartySrc\computecpp-sdk\build\samples\gaussian-blur\gaussian-blur.obj	1	
```

~~Linking against ComputeCpp_vs2015.lib does not reproduce the issues though~~

EDIT: I take back what I just said. Linking against ComputeCpp_vs2015.lib in debug build still produces the same "unresolved external symbol" error. Release build is fine.

**Comment by rodburns** [comment]
Thanks for the report. We are working on a fix for this issue.

**Comment by rodburns** [comment]
This is now fixed in the 1.1.0 release.
